src/org/rascalmpl/library/IO.rsc

```diff
@@ -362,6 +369,22 @@ created(|std:///IO.rsc|);
 @javaClass{org.rascalmpl.library.Prelude}
 public java datetime created(loc file);
 
+@synopsis{File size of a location.}
+@description{
+Returns the file size of the file at location `file`.
+}
+@examples{
+```rascal-shell
+import IO;
+```
+Determine the file size of the Rascal standard library:
+```rascal-shell,continue
+fileSize(|std:///IO.rsc|);
+```
+}
+@javaClass{org.rascalmpl.library.Prelude}
+public java int fileSize(loc file);
```

Comment:
or should it be `size`? (even though that overlaps with set/list/map containers)

Reply:
it's not always a file. it can be a part of a file with the offset/length part of a loc. Also via logical resolvers it can be a part of a file without showing a offset/length explicitly.

Reply:
I think it should be `size` for the sake of consistency also, with Map, Set, List.

Reply:
hmm, yes the relation to length is a bit confusing.. as right this function would return the byte size, which might have a complex relation to `.length` of a location.

Reply:
I've dropped it. We've always prevented this from being exposed to end users, and we should keep it that way.